testing/tests/test_e2e/test_pebble.py

```diff
@@ -143,16 +145,18 @@ def callback(self: CharmBase):
         file = tmp_path / 'bar' / 'baz.txt'
 
         # another is:
-        assert file == Path(out.get_container('foo').mounts['foo'].source) / 'bar' / 'baz.txt'
+        assert (
+            file == pathlib.Path(out.get_container('foo').mounts['foo'].source) / 'bar' / 'baz.txt'
+        )
 
         # but that is actually a symlink to the context's root tmp folder:
         assert (
-            Path(ctx._tmp.name) / 'containers' / 'foo' / 'foo' / 'bar' / 'baz.txt'
+            pathlib.Path(ctx._tmp.name) / 'containers' / 'foo' / 'foo' / 'bar' / 'baz.txt'
         ).read_text() == text
         assert file.read_text() == text
 
         # shortcut for API niceness purposes:
-        file = container.get_filesystem(ctx) / 'foo' / 'bar' / 'baz.txt'
+        file = container.get_filesystem(ctx) / 'foo' / 'bar' / 'baz.txt'  # type: ignore
```

Comment:
Why is `type: ignore` needed here?

Reply:
I think the signature for `get_filesystem` needs to take a `Context[ops.CharmBase]` rather than plain `Context` or something like that.